src/main/java/frc/robot/subsystems/climber/ClimberIO.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.subsystems.climber;
+
+import org.littletonrobotics.junction.AutoLog;
+
+/** Generic subsystem hardware interface. */
+public interface ClimberIO {
+  /** Contains all of the input data received from hardware. */
+  @AutoLog
+  public static class ClimberIOInputs {
+    double leftVelocityRPM = 0.0;
+    double leftPosition = 0.0;
```

Comment:
Specific units. It may be helpful to have the units in meters relative to the top of the winch....